packages/app/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@dmapper/app",
+  "version": "2.0.1",
```

Comment:
Version should be managed by lerna. Initially set it to be the same as current version we have in lerna and then do that command in lerna where it goes through each package and checks if it's published or not -- which should publish @startupjs/app